docs/api/unified_api/introduction.rst

```diff
@@ -5,6 +5,7 @@ Introduction
 
     The new, unified Sylius API is still under development, that's why the whole ``ApiBundle`` is tagged with ``@experimental``.
     This means that all code from ``ApiBundle`` is excluded from :doc:`Backward Compatibility Promise </book/organization/backward-compatibility-promise>`.
+    You can disable entire API by changing the flag ``sylius_api.enabled`` to ``false`` in ``src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml``.
```

Comment:
I guess that we should have API disabled by default:
```suggestion
    You can enable entire API by changing the flag ``sylius_api.enabled`` to ``true`` in ``app/config/packages/_sylius.yaml``.

    .. code-block:: yaml

        # app/config/packages/_sylius.yaml
        sylius_api:
            enabled: true
```

Reply:
And if the API will be disabled by default, there should be a note in the UPGRADE file